nessai/proposal/flowproposal.py

```diff
@@ -801,6 +801,11 @@ def verify_rescaling(self):
             raise RuntimeError(
                 "Rescaling must be set before it can be verified"
             )
+        if not self._reparameterisation.one_to_one:
+            logger.warning(
+                "Could not check if reparameterisation is invertible"
```

Comment:
Could not check that it is invertible, or checked and found that it is not invertible?

Reply:
At the moment, it's implemented as could not check since it should always fail but if `one_to_one=True`, it will check and raise an error if they're not invertible.